<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormGoodsKey700</class>
 <widget class="QWidget" name="FormGoodsKey700">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#FormGoodsKey700{border-image: url(:/image/report.png);}</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>381</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>66</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>70</number>
   </attribute>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>新建行</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>第1列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>第2列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>第3列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>第4列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>第5列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>第6列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>第7列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>第8列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>第9列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>第10列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>第11列</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>第12列</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="lineEdit_goodsCode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_keys">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>390</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_msg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>621</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_print">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>440</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>打印商品编码明细</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
